### PR TITLE
Update streamlit version

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Small PR, one reviewer needed 🦺 
Once this is merged to dev, then we can merge dev to master, address the dependabot issues, and deploy

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -5,13 +5,14 @@ description = ""
 authors = ["Sasha Weinstein <70385853+SashaWeinstein@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.9.7 || >3.9.7,<4.0"
```
Comment (mbh329) [inline pyproject.toml:8]: Why do we have to be this specific with the version of python we are using? I see that in .toml files for other data products we are this specific, just curious if this is automatically updated by poetry or if this is something you had to specify

File: pyproject.toml [hunk 2]
```diff
@@ -5,13 +5,14 @@ description = ""
...
 pandas = "^1.4.3"
 numpy = "^1.23.1"
-streamlit = "plotly"
+
 matplotlib = "^3.5.2"
 python-abc = "^0.2.0"
 numerize = "^0.12"
+streamlit = "^1.13.0"
```
Comment (mbh329) [inline pyproject.toml:15]: This is the most recent version of streamlit, dope!

File: pyproject.toml [hunk 3]
```diff
@@ -5,13 +5,14 @@ description = ""
 authors = ["Sasha Weinstein <70385853+SashaWeinstein@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.9.7 || >3.9.7,<4.0"
 pandas = "^1.4.3"
 numpy = "^1.23.1"
-streamlit = "plotly"
```
Comment (mbh329) [inline pyproject.toml:11]: Were you able to get to the bottom of why  this happened?

Comment (SashaWeinstein) [reply pyproject.toml:11]: lol no. I didn't look that hard though, could comb through commit history more carefully

File: pyproject.toml (hunk 1 above)
Comment (SashaWeinstein) [reply pyproject.toml:8]: It's a streamlit dependency thing in my understanding. I think the issue is specifically with 3.9.7, this versioning avoids that release

**Pull request merged**